docs/reference/release.md

```diff
@@ -28,17 +68,23 @@ git push origin v3.1.1
 ```
 4. Create a merge PR against master that only bumps the version number in `gutenberg.php`, `package.json`, and `package-lock.json`.
 
-## Build the Release
+### Build the Release
 
 Note: The `1.x.0` notation `git` and `svn` commands should be replaced with the version number of the new release.
 
 * Run `git checkout master` and `git pull`. Make sure your local `master` is up to date; you can confirm this by opening `gutenberg.php` and checking for the version bump you merged previously.
```

Comment:
This is no longer true, because we can't build from master. We need to build from the last RC instead and we need to commit a version bump commit on the branch holding the last RC (or just locally above the tag) and also merge this commit back to master.

Reply:
Oh snap, I thought this and figured I should update. I'll do another patch, didn't expect this to get merged so quickly! 😅